p3-farm-nft/P3_nft_explanation.md

```diff
@@ -0,0 +1,74 @@
+# P3 (NFT versioned) explanation 
```

Comment:
shall we call it `p4-farm-nft`? We already have p3 farm, and it's working totally differently.